### ISSUE TITLE
Example of certifierCredential property

### ISSUE THREAD

**Issue by LaurenceZaysser** [mattgarrish/epub-revision#2760]
### Section

https://www.w3.org/TR/epub-a11y-11/#sec-evaluator-credentials

### Describe the problem

The example does not tell anything about the credentials/competency evidences of the evaluator.
It is neither a URL to a web page describing its credentials, nor the raw text of his credentials.
This is very misleading for people who cannot understand from abstract definitions (most of digital project managers are like this in the publishing area, as they have NO background in computer science), but only from true examples.

### Describe the fix or new feature you propose

```
<meta
       property="a11y:certifierCredential"
       refines="#certifier">
      All people responsible for accessibility assessment in this company have been trained in WCAG and associated control tools.
   </meta>
```

**Comment by mattgarrish** [comment]
A credential is issued from a known authority. Benetech offers their GCA credential to publishers, for example. The example in the specification is intentionally vague as we don't use real examples to avoid showing preference for any one organization.

Users aren't likely to have difficulty looking up the credential, and in some cases they may have linked access to more information (but that's a metadata display issue).

**Comment by LaurenceZaysser** [comment]
Yes indeed, I misunderstood the property, as the definition was not that clear. Sorry for disturbing.
[https://www.w3.org/TR/epub-a11y-11/#certifierCredential](https://www.w3.org/TR/epub-a11y-11/#certifierCredential)
The last ONIX version of code list 196 defines it in a clearest way if I may.
[https://ns.editeur.org/onix/en/196/89](https://ns.editeur.org/onix/en/196/89)
[https://ns.editeur.org/onix/en/196/90](https://ns.editeur.org/onix/en/196/90)

What about a nice/funny name for an  independant organization then? :-)
- _UnitedImpairedOfTheWorld  Association_
- _DontTrustCheckA11Y Association_
- ...

**Comment by mattgarrish** [comment]
Not for this group to do, but it might be helpful to keep a list of information for displaying credentials as part of the metadata display guide work, especially if we go with limiting the metadata to text strings. The image, URL, alt text, etc. to display in place of the text could be made available to anyone implementing the guide rather than have Benetech et al. have to reach out to each implementer.

**Comment by clapierre** [comment]
I was asked to provide details on how Benetech and our Global Certified Accessible (GCA) program is using these metadata fields.
Let me start off by saying that since the W3C does not want to specifically endorse a company / program / tool this is just for providing clarity on how this is actually being used in the industry and if we want to create some examples in our documents we can make up some fake names to demonstrate the concepts to help publishers understand when and how this metadata.

Benetech partnered with a few companies, and we developed a process for rigorous testing using both automated and manual inspection and testing including the use of "EPUBCheck, ACE by DAISY, SMART by DAISY, GCA-SMART (which includes an additional 90+ checks over and above the WCAG requirements.

Our GCA process also requires yearly re-certification which must pass the WCAG 2.2 AA compliance, and in addition each quarter we update our process with the latest standards and best practices as set by the industry.

GCA certifies publisher workflows and that the EPUBs coming from those workflows meet the EPUB Accessibility 1.1 Specification at WCAG 2.2 AA level of conformance.

For a publisher who has passed GCA certification they can add the following GCA metadata to their EPUBs in addition to the accessibilityFeatures, accessibilityHazards, accessModes, accessModeSufficient, and accessibilitySummary metadata.

### Benetech Direct GCA Certified
```
<meta property="dcterms:conformsTo" id="conf"> EPUB Accessibility 1.1 - WCAG 2.2 Level AA </meta> 
<meta property="a11y:certifiedBy" refines="#conf" id="certifier">Benetech</meta> 
<meta property="a11y:certifierCredential" refines="#certifier"> (url link to Benetech's GCA credentials page) </meta>
<meta property="dcterms:date" refines="#certifier"> yyyy-mm-dd </meta>

```
### Benetech Partner Using our Credentials - GCA Certified
Benetech has also partnered with EBound Canada, where we trained them on our GCA certification process and have validated their results. in this case they use our certifierCredential but the certifiedBy is slightly different.

```
<meta property="dcterms:conformsTo" id="conf"> EPUB Accessibility 1.1 - WCAG 2.2 Level AA </meta> 
<meta property="a11y:certifiedBy" refines="#conf" id="certifier">Benetech via eBound Canada</meta> 
<meta property="a11y:certifierCredential" refines="#certifier"> (url link to Benetech's GCA credentials page) </meta>
<meta property="dcterms:date" refines="#certifier"> yyyy-mm-dd </meta>
```

**Comment by clapierre** [comment]
I think it would also be useful to provide an example of when 3rd party certification is not provided, and an individual author just runs the automatic checks like ACE what should they include in their metadata.

In that case we don't want to see any conformance claim to the EPUB Accessibility 1.1 Specification or to a specific WCAG conformance level and what might be appropriate here would be to add something to the effect in the accessibilitySummary "This EPUB passes ACE by DAISY version 1.3.6."

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 22 August 2025](https://w3c.github.io/pm-wg/minutes/2025-08-21-a11y.html#ecd6).

<details><summary><i>View the transcript</i></summary>

<h3>Example of certifierCredential property: Issue #2760</h3>
<p><cite>&lt;AvneeshSingh&gt;</cite> <a href="https://github.com/w3c/epub-specs/issues/2760" rel="noopener noreferrer">w3c/<wbr>epub-specs#2760</a></p>
<p><cite>CharlesL:</cite> it's about W3C not endorsing companies. That mean we have to create a fake webpage to point to in an example.</p>
<p><cite>CharlesL:</cite> I can use the real information in the issue and let Matt do the fake example from it.</p>
<p><cite>DaleRogers:</cite> A question, I finished a book, went into all validators, I published the report and added the link to it in the ebook. The notion of being certified, I am not sure if it is about US policy and could be different in EU per example.</p>
<p><cite>CharlesL:</cite> you are right, we need to figure out who did the evaluation. The certifier credential is refering to a defiined process.</p>
<p><cite>AvneeshSingh:</cite> it's brand image and reputation thing.</p>
<p><cite>DaleRogers:</cite> As an independant content creator, I can go to the process, but Certify sound Official. I don't feel confident to use it. I want to document the efforts I made to make it accessible. But I don't want to enter into claming something.</p>
<p><cite>CharlesL:</cite> in your case, what you could do is to use the accessibility summary to describe your process.</p>
<p><cite>CharlesL:</cite> It's interesting, we could probably make this clearer about what process is minimum required.</p>
<p><cite>DaleRogers:</cite> I would apreciate guidance oabout what i can claim and what i cannot.</p>
<p><cite>George:</cite> third party certification is usfull to make sure the reports are well done, features well tested.</p>

<hr /></details>

**Comment by rickj** [comment]
I'm concerned we are over complicating this.  We have `a11y:certifiedBy` already, where someone can declare who did any certification.  Then it becomes a matter of how the marketplace establishes trust for that declaration.  Even without an `a11y:certifierCredential` present, the marketplace will establish trust by  `a11y:certifiedBy` being the publisher itself, or an external service, or someone never heard of (with resulting lower trust, I would imagine!)

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 04 September 2025](https://w3c.github.io/pm-wg/minutes/2025-09-04-a11y.html#28ea).

<details><summary><i>View the transcript</i></summary>

<h3>Example of certifierCredential property: Issue #2760</h3>
<p><cite>&lt;AvneeshSingh&gt;</cite> <a href="https://github.com/w3c/epub-specs/issues/2760" rel="noopener noreferrer">w3c/<wbr>epub-specs#2760</a></p>
<p><cite>mgarrish:</cite> are we at a point to make changes with this?<br>
<span>… is it a URL or text is it only URLs or it</span></p>
<p><cite>Charles:</cite> in GCA's case the URL is our credential and thousands of books has this as the certifierCredential. Vitalsource does look for this to put our GCA logo.</p>
<p><cite>AvneeshSingh:</cite> if it is text, and we put a URL in it, it doesn't prevent anyone to actually parse the URL and provide the logo or provide a linked URL.</p>
<p><cite>mgarrish:</cite> the issue is it should have a Human Readable name. instead of the ugly URL.<br>
<span>… , we don't need a registry of what URLs are out there.  Do we need to change anything?</span></p>
<p><cite>George:</cite> Bookshare puts in human readable URL.</p>
<p><cite>mgarrish:</cite> it will be up to the agency what should be in this field. we are open and doesn't prefer anything.</p>
<p><cite>AvneeshSingh:</cite> the demand here is show some examples. So, we can show example with text. And we can place a note that if you want to use a URL then it is your responsibility to reach out to the distributors to display something else like a logo.</p>

<hr /></details>
